kratos/includes/mortar_classes.h

```diff
@@ -945,21 +954,21 @@ class MortarOperator
     {
         // We calculate the inverse of D operator
         double auxdet;
-        const boost::numeric::ublas::bounded_matrix<double, TNumNodes, TNumNodes> InvDOperator = MathUtils<double>::InvertMatrix<TNumNodes>(DOperator, auxdet);
+        const boost::numeric::ublas::bounded_matrix<double, TNumNodes, TNumNodes>& inv_D_operator = MathUtils<double>::InvertMatrix<TNumNodes>(DOperator, auxdet);
```

Comment:
minor: i think it will work if you omit the "boost::numeric ..."

Reply:
Yep